internal/generator/vector/output/otlp/group_by.go

```diff
@@ -43,7 +43,7 @@ func GroupByContainer(id string, inputs []string) Element {
 		Inputs:      helpers.MakeInputs(inputs...),
 		MaxEvents:   "250",
 		GroupBy: MakeGroupBys(".openshift.cluster_id",
-			".kubernetes.namespace_name", ".kubernetes.pod_name", ".kubernetes.container_name"),
+			".kubernetes.namespace_name", ".kubernetes.pod_name", ".kubernetes.container_name", ".kubernetes.container_iostream"),
```

Comment:
@vparfonov iostream here is not an attribute to group_by for OTLP since its simply an attribute of the log entry and not the resource which is the container